docs/semgrep-ci/network-broker.md

```diff
@@ -0,0 +1,134 @@
+---
+slug: network-broker
+title: Semgrep Network Broker
+hide_title: false
+description: Learn how to set up the Semgrep Network Broker, which facilitates secure access between Semgrep and your private network.
+tags:
+  - Semgrep in CI
+  - Network broker
+---
+
+# Set up the Semgrep Network Broker
+
+The Semgrep Network Broker facilitates secure access between Semgrep and your private network. It accomplishes this by establishing a WireGuard VPN tunnel with the Semgrep infrastructure, then proxying **inbound** HTTP requests from Semgrep to your network through this tunnel. This approach allows Semgrep to interact with on-premise resources without exposing them to the public internet.
+
+Examples of inbound traffic include:
+
+- Pull request (PR) or merge request (MR) comments
+- Jira integrations
+- Webhooks
+
+:::info Subscriber availability
+The Semgrep Network Broker is available to Enterprise tier users.
+:::
+
+## Prerequisites
+
+- The Semgrep Network Broker is a feature that must be enabled in your Semgrep organization (org) before set up. Contact the Semgrep team on [<i class="fas fa-external-link fa-xs"></i> Slack]( https://go.semgrep.dev/slack) or through email [<i class="fa-regular fa-envelope"></i> support@semgrep.com](mailto:support@semgrep.com).
```

Comment:
```suggestion
- The Semgrep Network Broker is a feature that must be enabled in your Semgrep organization (org) before setup. It is only available to paying customers. Contact the [Semgrep support team](/docs/support) to discuss having it enabled for your organization.
```
Pending checking on who this can be enabled for, but definitely want to say who and link to the support page.

Reply:
Seems like paying customer is probably fine for now 👍🏻